scripts/travis/travis-tests

```diff
@@ -19,6 +19,35 @@ if [ "$UNAME_M" != "x86_64" ]; then
 	SKIP_TRAVIS_TEST=1
 fi
 
+install_retry_counter=0
+max_apt_retries=10
```

Comment:
Do you think a smaller number of retries won't be sufficient?

Reply:
Initially I had 5 retries. 10 is indeed a lot and if it does not work after the third or fourth it will probably fail the tenth time.

I can change it to 5. Maybe I should add a `sleep $install_retry_counter` to give it some time to recover from errors.

Reply:
done